src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReaderServiceImplTest {
+    @Test
```

Comment:
```suggestion
    @Test(expected = RuntimeException.class)
```
this way you can expect error to occur in your test